fontforgeexe/startui.c

```diff
@@ -1235,8 +1236,10 @@ int fontforge_main( int argc, char **argv ) {
     for ( i=1; i<argc; ++i ) {
 	char *pt = argv[i];
 
-	if ( !strcmp(pt,"-SkipPythonInitFiles")) {
-	    ProcessPythonInitFiles = 0;
+	if ( strcmp(pt,"-SkipPythonInitFiles")==0 ) {
```

Comment:
I do mention `-SkipPythonPlugins` elsewhere in the documentation but, yes, I didn't add them to the help string because the parallel I chose wasn't there either. Should I just add them? Seems easy enough. Probably need to go into the man page as well. 

If someone could give me phrasing for `-SkipPythonInitFiles` I would appreciate it. I never no how much detail to go into.

We could also change one or both to match stylistically *given* that the existing one is undocumented although doing that with `-SkipPythonInitFiles` might cause issues.

Reply:
I changed these to `-skippyfile` and `-skippyplug` and documented them, leaving `-SkipPythonInitFiles` as an undocumented alternative to the former. I also cleaned up the `-help` messages a bit. 

(So by *grepping* I found the command line documentation in a *techref*, listed alphabetically, because that make so much sense--who wouldn't look there? If only it were possible to do more with the documentation than throw it in a dumpster and light it on fire ...)